### PR TITLE
Fixes Emissives, aswell as adds an atomized version of update_appearance 

### PR DESCRIPTION
**Pull request by dwasint**
## About The Pull Request
Readds emissives, also converts emissives to tg's standard of emissive
mostly ports: https://github.com/tgstation/tgstation/pull/58130
aswell as a portion of: https://github.com/tgstation/tgstation/pull/55468

## Why It's Good For The Game
Readds a feature disabled on accident 

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

image

</details>

## Changelog
:cl:
fix: computers now glow again
/:cl:

### CODE REVIEW

**Comment by itsmeow** [comment]
Does this also work on fire alarms and APCs?

**Comment by dwasint** [comment]
will do those aswell give me like 2 seconds

**Comment by dwasint** [comment]
> Does this also work on fire alarms, APCs, and vending machines?

they are emissive its just they use the old version, fire alarms were updated apcs will take a few minutes

**Comment by itsmeow** [comment]
wow, saving me 0.18s. instant merge

**Comment by Archanial** [comment]
I need this so much

File: code/modules/hydroponics/gene_modder.dm [hunk 1]
```diff
@@ -449,7 +449,8 @@
 	src.pixel_x = rand(-5, 5)
 	src.pixel_y = rand(-5, 5)
 
-/obj/item/disk/plantgene/proc/update_name()
+/obj/item/disk/plantgene/update_name()
+	.=..()
```
Comment (PowerfulBacon) [inline code/modules/hydroponics/gene_modder.dm]:
```suggestion
	. = ..()
```

File: code/modules/mob/living/simple_animal/bot/bot.dm [hunk 2]
```diff
@@ -934,6 +934,7 @@ Pass a positive integer as an argument to override a bot's default speed.
 	update_controls()
 
 /mob/living/simple_animal/bot/update_icon_state()
+	.=..()
```
Comment (PowerfulBacon) [inline code/modules/mob/living/simple_animal/bot/bot.dm]:
```suggestion
	. = ..()
```

File: code/modules/mob/living/simple_animal/slime/slime.dm [hunk 3]
```diff
@@ -129,7 +129,8 @@
 	coretype = text2path("/obj/item/slime_extract/[sanitizedcolour]")
 	regenerate_icons()
 
-/mob/living/simple_animal/slime/proc/update_name()
+/mob/living/simple_animal/slime/update_name()
+	.=..()
```
Comment (PowerfulBacon) [inline code/modules/mob/living/simple_animal/slime/slime.dm]:
```suggestion
	. = ..()
```

File: code/modules/paperwork/paper.dm [hunk 4]
```diff
@@ -109,6 +109,7 @@
 	update_icon()
 
 /obj/item/paper/update_icon_state()
+	.=..()
```
Comment (PowerfulBacon) [inline code/modules/paperwork/paper.dm]:
```suggestion
	. = ..()
```

File: code/modules/power/lighting.dm [hunk 5]
```diff
@@ -372,6 +372,7 @@
 			icon_state = "[base_state]-burned"
 		if(LIGHT_BROKEN)
 			icon_state = "[base_state]-broken"
+	return ..()
```
Comment (PowerfulBacon) [inline code/modules/power/lighting.dm]:
```suggestion
	. = ..()
```

File: code/modules/reagents/reagent_containers/chem_heirloom.dm [hunk 6]
```diff
@@ -30,7 +30,8 @@
 		smartguy = TRUE
 	. += "It contains:\n100 units of [smartguy ? initial(rand_cont.name) : "various reagents"]" //Luckily science goggles say nothing if there's no reagents
 
-/obj/item/reagent_containers/glass/chem_heirloom/proc/update_name() //This has to be done after init, since the heirloom component is added after.
+/obj/item/reagent_containers/glass/chem_heirloom/update_name() //This has to be done after init, since the heirloom component is added after.
+	.=..()
```
Comment (PowerfulBacon) [inline code/modules/reagents/reagent_containers/chem_heirloom.dm]:
```suggestion
	. = ..()
```

**Review by PowerfulBacon — COMMENTED** [review]
Great changes, definitely will be playing around with these for some nifty visuals.

**Comment by dwasint** [comment]
out of scope for this pr, but with emissives working again i'd recommend looking into the light radius of computers and such to be darker,
My pr on monkey has them set to half the current light

**Comment by itsmeow** [comment]
not really out of scope, you can just change the number

**Comment by dwasint** [comment]
> not really out of scope, you can just change the number

image

left is old right side is new

**Comment by itsmeow** [comment]
Looks great in game on local, although emissives now show through multi-z.. RIP. I guess we need plane cubes. Seems fine to me.

**Comment by dwasint** [comment]
> Looks great in game on local, although emissives now show through multi-z.. RIP. I guess we need plane cubes. Seems fine to me.

yea the multi-z support was only added in the plane cube pr, which is a whole other thing that needs to be done as we need all the plane changes from tg

**Review by itsmeow — APPROVED** [review]
Works really well on local

**Comment by PowerfulBacon** [comment]
Awesome, definitely will look into making use of these

**Pull request merged**